### PR TITLE
Updated wishlist analytics readme file

### PR DESCRIPTION
**Pull request by chiranjeevi-cj**
### Description (*)

Update README.md file at Magento_WishlistAnalytics module.

### Fixed Issues (if relevant)

1. Fixes magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by chiranjeevi-cj** [comment]
@magento run all tests

File: app/code/Magento/WishlistAnalytics/README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+## Installation details
+
+Before installing this module, note that the **Magento_WishlistAnalytics** is dependent on the following modules:
+
+- Magento_Analytics
+- Magento_Wishlist
+
+For information about enabling or disabling a module in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+
+## Extensibility
```
Comment (orlangur) [inline app/code/Magento/WishlistAnalytics/README.md]: Where do you see it? https://github.com/magento/magento2/tree/2.4-develop/app/code/Magento/WishlistAnalytics

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/WishlistAnalytics/README.md (hunk 1 above)
Comment (chiranjeevi-cj) [reply app/code/Magento/WishlistAnalytics/README.md]: @orlangur I thought that was the general topic. I removed that content. Thanks!

**Comment by chiranjeevi-cj** [comment]
@magento run all tests

**Comment by magento-engcom-team** [comment]
Hi @chiranjeevi-cj
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31369 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Pull request closed without merging**